easybuild/easyconfigs/k/Kent_tools/Kent_tools-337-foss-2016a.eb

```diff
@@ -0,0 +1,31 @@
+#This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#Author: Ravi Tripathi
+#Email: ravi89@uab.edu
+
+easyblock = "PackedBinary"
+
+name = 'Kent_tools'
+version = '337'
+
+homepage = 'http://genome.cse.ucsc.edu/'
+description = """Kent tools: collection of tools used by the UCSC genome browser."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://hgdownload.soe.ucsc.edu/admin/exe/']
+sources = ['userApps.v%(version)s.src.tgz']
+
+checksums = ['f72b858b544f0754c733cc1413acf284']
+
+dependencies = [
+        ('libpng', '1.6.23'),
+]
+
+install_cmd = "cp -r %(builddir)s/userApps/* %(installdir)s && cd %(installdir)s && make"
```

Comment:
also, this also copies all the source code to the installation directory, which is maybe not what we want...

how about using the `MakeCp` easyblock instead?